content/pages/edit.json

```diff
@@ -44,6 +44,13 @@
         "options": ["Forever", "1 day", "1 week", "1 month", "1 year"],
         "sortOptions": false,
         "required": true
+      },
+      {
+        "name": "author",
+        "label": "New Author",
+        "placeholder": "e.g. Mrs McJellyfish",
+        "help": "Give proper attribution for your data set.",
+        "required": true
```

Comment:
here I'm not sure how Formik behaves with this to cover this use case:

> publisher can remove the author as part of _Edit Metadata_ flow

So would clearing the field still make it submittable or would form error be shown?

Reply:
Formik won't let you submit an empty form field for the author as it is a required field. The submit button is disabled if you clear the author field